contracts/babylon/src/msg/ibc.rs

```diff
@@ -48,6 +47,7 @@ pub struct BsnRewards {
 
 #[cw_serde]
 pub struct FpRatio {
-    pub btc_pk: Binary,
+    // FP BTC public key, encoded in hex
+    pub btc_pk: String,
```

Comment:
We may consider a unit type wrapper or simply an alias, something like `BtcPubKey` for the BTC public key encoded in hex string, so that it's super clear what it's from the type itself. Can be done in the future.

Reply:
Usually there's a convention that a hex field name ends in `_hex`.